packages/react-app/src/components/filter.tsx

```diff
@@ -61,6 +64,11 @@ export function Filter({ compact }: Props) {
   const { below } = useViewport();
   const states = [ENUM_QUEST_STATE.All, ENUM_QUEST_STATE.Active, ENUM_QUEST_STATE.Expired];
   const { isFilterShown } = useFilterContext();
+  const isFilteringOriginalState =
```

Comment:
I'd make a useEffect and compare actual filter value with the default one in constant constants.ts->DEFAULT_FILTER

Reply:
So if we change the default config, no need to adapt this code

Reply:
true, good idea